server/src/test/java/io/crate/execution/engine/collect/collectors/LuceneBatchIteratorTest.java

```diff
@@ -38,16 +38,17 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import io.crate.expression.reference.doc.lucene.CollectorContext;
+import io.crate.expression.reference.doc.lucene.LongColumnReference;
+import io.crate.testing.BatchIteratorTester;
 
-public class LuceneBatchIteratorTest extends ESTestCase {
+public class LuceneBatchIteratorTest {
```

Comment:
The `LuceneTestCase` base class still randomly injects some assertions via the `AssertingDocValuesFormat` (which is loaded dynamically via the codec mechanism)

Given that we have few unit tests which access doc-values from different threads I think this might be a acceptable option to remove it from the base classes?